src/web/components/App.tsx

```diff
@@ -77,9 +78,24 @@ export const App = ({ CAPI, NAV }: Props) => {
         //
         // Note: Both require a 'root' element that needs to be server rendered.
         <>
+            <Portal root="reader-revenue-links-header">
+                <ReaderRevenueLinks
+                    urls={CAPI.nav.readerRevenueLinks.footer}
+                    edition={CAPI.editionId}
+                    dataLinkNamePrefix="footer : "
+                    noResponsive={true}
+                    inHeader={false}
+                />
+            </Portal>
             <Hydrate root="links-root">
                 <Links isSignedIn={isSignedIn} />
             </Hydrate>
+            <Hydrate root="edition-root">
+                <EditionDropdown
+                    edition={CAPI.editionId}
+                    dataLinkName="nav2 : topbar : edition-picker: toggle"
+                />
+            </Hydrate>
```

Comment:
Again, this was just me moving things around so the order was more logical to read